examples/language-analyzer.go

```diff
@@ -26,14 +26,33 @@ var version = "alpha"
 func (*analyzer) NotifyReviewEvent(ctx context.Context, review *pb.ReviewEvent) (*pb.EventResponse, error) {
 	log.Infof("got review request %v", review)
 
-	conn, err := pb.DialContext(ctx, dataSrvAddr)
+	eh := func(err error, msg string) {
+		log.Errorf(err, msg)
+	}
+	conn, err := pb.DialContext(
+		ctx,
+		dataSrvAddr,
+		grpc.WithStreamInterceptor(grpchelper.CtxlogStreamClientInterceptor(eh)),
+		grpc.WithUnaryInterceptor(grpchelper.CtxlogUnaryClientInterceptor(eh)),
+	)
+
 	if err != nil {
 		log.Errorf(err, "failed to connect to DataServer at %s", dataSrvAddr)
 		return nil, err
 	}
 	defer conn.Close()
 
 	dataClient := pb.NewDataClient(conn)
+
+	// Add some log fields that will be available to the data server
+	// using `pb.UpdateLogFields`.
+	// It's also possible to read, set and delete log fields using
```

Comment:
not sure if we should allow to change fields sent by lookout. Reading/adding most probably yes, but deleting/changing... Carlos wdyt?

Reply:
mhmm... well if an analyzer would like to delete/change some keys it could actually do it anyway, but maybe we shouldn't provide an easy api for it and write somewhere that this is discouraged.

Reply:
yes. I'm talking about sdk API level.

Reply:
done.